chef/data_bags/crowbar/migrate/network/021_add_mtu_os_sdn.rb

```diff
@@ -0,0 +1,15 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"]["os_sdn"].key? "mtu"
+    a["networks"]["os_sdn"]["mtu"] = ta["networks"]["os_sdn"]["mtu"]
+  end
+
+  return a, d
+end
+
+def downgrade(ta, td, a, d)
+  unless a["networks"]["os_sdn"].key? "mtu"
```

Comment:
When you downgrade, ta comes from the old template. Which shouldn't include the key (except if for some reason, it's still there).

TBH, the downgrade bits are probably not very well working ;-) But the intent here is definitely to check the template.